bin/GameIndex.dbf

```diff
@@ -1735,10 +1734,10 @@ Region = PAL-M8
 Compat = 5
 XgKickHack = 1
 [patches = 80802EA9]
-comment= Patch By Prafull
-// fix hang at loading screen
-patch=1,EE,002aa934,word,00000000
-patch=1,EE,002aa574,word,00000000
+	comment= Patch By Prafull
+	fix hang at loading screen
```

Comment:
You removed the `//` here. You should also add a `.` at the end of the line like the other changes.